src/commonTest/kotlin/org/cru/godtools/tool/model/ManifestTest.kt

```diff
@@ -0,0 +1,58 @@
+package org.cru.godtools.tool.model
+
+import org.cru.godtools.tool.internal.AndroidJUnit4
+import org.cru.godtools.tool.internal.RunOnAndroidWith
+import org.cru.godtools.tool.internal.TEST_XML_PULL_PARSER_FACTORY
+import org.cru.godtools.tool.internal.UsesResources
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+
+@RunOnAndroidWith(AndroidJUnit4::class)
+class ManifestTest : UsesResources {
+    override val resourcesDir = "model"
+
+    // region parse Manifest
+    @Test
+    fun testParseManifestEmpty() {
+        val manifest = parseManifest("manifest_empty.xml")
+        assertNull(manifest.title)
+        assertEquals(Manifest.Type.DEFAULT, manifest.type)
+//        assertEquals(DEFAULT_LESSON_CONTROL_COLOR, manifest.lessonControlColor)
+//        assertEquals(DEFAULT_TEXT_SCALE, manifest.textScale, 0.0001)
+//        assertEquals(0, manifest.aemImports.size)
+//        assertThat(manifest.lessonPages, `is`(empty()))
+//        assertThat(manifest.tractPages, `is`(empty()))
+//        assertEquals(0, manifest.resources.size)
+//        assertEquals(0, manifest.tips.size)
```

Comment:
yeah, I copied the test from godtools, and have been enabling checks as support is added. Right now I only enabled a few things to cover the core logic of the XML parser itself. More of this will be fleshed out in subsequent PRs, I just wanted to try and avoid a gigantic PR covering everything